earth_enterprise/src/fusion/rasterfuse/TmeshGenerator.cpp

```diff
@@ -74,14 +77,14 @@ TmeshGenerator::Generate(float *srcSamples,
   elev.resize(SizeX * SizeY);
 
   // Error thresholds for decimation - set these to 0 to skip decimation
-  // jdjohnso - These numbers came from empirical testing?
-  // jdjohnso - The allowable error gets smaller as the resolution increases.
-  // jdjohnso - The allowable error on border tiles is 1/10 that of interior
+  // Question - Did these numbers come from empirical testing?
+  // The allowable error gets smaller as the resolution increases.
+  // The allowable error on border tiles is 1/10 that of interior
   // tiles. This is because we really want the border tiles to match. If not
   // we get rips between mesh tiles in the client.
-  int error_level = tmeshAddr.level+1;
-  double error = 0.009 * pow(2, 11-error_level);
-  double borderError = error / 10.0;
+  double threshold_modifier = pow(2, 11 - (tmeshAddr.level + 1));
```

Comment:
If "(tmeshAddr.level + 1)" computes an 'error_level', I would restore the variable 'error_level' as it was. This way we would follow how 'thereshold_modifier' is being calculated using an 'error_level' instead of having one complex equation not easy to follow.

Reply:
I disagree.  I am already using a separate variable to hold threshold_modifier, it seems very easy to understand.  The previous way, with error = 0.009 * pow(2, 11-error_level)   seems much more complicated.

Reply:
I mean at that point you have to wonder why 2, why 11, why divide by 10.0

Reply:
Yes it's already convoluted, that is why i suggested to make it less by restoring 'error_level'. It will not explain still the equation but it will at least keep the history that 'tmeshAddr.level+1' is some kind level of an error.